.deploy/naiserator.yaml

```diff
@@ -50,6 +50,7 @@ spec:
       rules:
         - application: familie-brev
         - application: familie-ef-blankett
+        - application: soknad-kontantstotte-api
```

Comment:
This is not really needed if `soknad-kontantstotte-api` runs on-prem

Reply:
Due to we are not using azure for auth for this app, and not running on gcp

Reply:
No, actually we are gonna move sokanad-kontantstotte-api to gcp:)